lighthouse-core/gather/driver.js

```diff
@@ -219,17 +220,19 @@ class Driver {
    * Evaluate an expression in the context of the current page.
    * Returns a promise that resolves on the expression's value.
    * @param {string} expression
+   * @param {{useIsolation: boolean}=} options
    * @return {!Promise<*>}
    */
-  evaluateAsync(expression) {
+  evaluateAsync(expression, options) {
+    const {useIsolation = true} = options || {};
```

Comment:
the double defaults here is kind of confusing. Maybe move the `{}` default for `options` to the params?

Reply:
done